src/plugins/recordTypes/iterationreport/forms/default.jsx

```diff
@@ -0,0 +1,172 @@
+            <Field name="installerExtent" />
+          </Field>
+        </Field>
+
+        <Field name="presenceGroupList">
+          <Field name="presenceGroup">
+            <Field name="installDeinstall" />
+            <Field name="artistPresent" />
+            <Field name="presentExtent" />
+          </Field>
+        </Field>
+
+        <Panel name="evaluation" collapsible collapsed>
+          <Cols>
+            <Col>
+              <Field name="evaluationGroupList">
+                <Field name="evaluationGroup">
+                  <Field name="iterationSuccessful" />
+                  <Field name="iterationEvaluator" />
+                </Field>
+              </Field>
+            </Col>
+
+            <Row>
+              <Field name="iterationViewed" />
+              <Field name="iterationApproved" />
+            </Row>
+          </Cols>
+
+          <Field name="iterationEvaluationNotes" />
+        </Panel>
+      </Panel>
+
+      <Panel>
+        <Panel name="space" collapsible collapsed>
+          <Field name="spaceGroupList">
+            <Field name="spaceGroup">
+              <Row>
+                <Field name="descriptionType" />
+                <Field name="approvalEntity" />
+                <Field name="approvalDate" />
+              </Row>
+              <Field name="spaceDescription" />
+            </Field>
+          </Field>
+        </Panel>
+      </Panel>
+
+      <Panel name="details" collapsible collapsed>
+        <Field name="exhibitionGroupList">
+          <Field name="exhibitionGroup">
+            <Field name="exhibitionCopyIdentificationNumber" />
+            <Field name="exhibitionApprovalEntity" />
+            <Field name="exhibitionApprovalDate" />
+          </Field>
+        </Field>
+
+        <Field name="installedEquipmentGroupList">
+          <Field name="installedEquipmentGroup">
+            <Field name="installedEquipmentDescription" />
+            <Field name="installedEquipmentApprovalEntity" />
+            <Field name="installedEquipmentApprovalDate" />
+          </Field>
+        </Field>
+
+        <Panel>
+          <Field name="technicalSetupGroupList">
+            <Field name="technicalSetupGroup">
+              <Row>
+                <Field name="technicalSetupType" />
+                <Field name="technicalSetupApprovalEntity" />
+                <Field name="technicalSetupApprovalDate" />
+              </Row>
+              <Field name="technicalSetupDescription" />
+            </Field>
+          </Field>
+        </Panel>
+
+        <Panel>
+          <Field name="iterationSpecificGroupList">
+            <Field name="iterationSpecificGroup">
+              <Row>
+                <Field name="modificationApprovalEntity" />
+                <Field name="modificationApprovalDate" />
+              </Row>
+              <Field name="modificationDescription" />
+            </Field>
+          </Field>
+        </Panel>
+
+        <Panel>
```

Comment:
This `<Panel>` was added outside of `<Field name="installationGroupList">`, but it should be inside `<Field name="installationGroup">`.

Current: (Note the unneccessary border and padding around "Further installation detail", and lack of padding around contents of "Further installation detail")
<img width="1297" alt="Screen Shot 2022-06-16 at 9 56 25 PM" src="https://user-images.githubusercontent.com/1395885/174208588-8d9eaed4-391a-4c86-bc59-1304a384cac2.png">

Desired:
<img width="1297" alt="Screen Shot 2022-06-16 at 9 59 29 PM" src="https://user-images.githubusercontent.com/1395885/174208637-274fb20b-013b-4ef1-a61c-902ae6765453.png">